spec/models/event_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe Event, type: :model do
+  subject {
+    Event.new(title: "Anything",
```

Comment:
kindly fix the alignment issue.

Reply:
set the indentation as per requirements.